packages/terra-alert/docs/MoreInformation.md

```diff
@@ -0,0 +1,142 @@
+|customStatusColor |string  |optional  |''  |The status bar color to be used for an alert of type custom.|
+|onDismiss |function  |optional  |null  |Callback function triggered when Dismiss button is clicked. The presence of this prop will cause the Dismiss button to be included on the alert.|
+|title |string  |optional  |''  |The title for the alert which will be bolded.|
+|type  |string  |optional  |'alert' |The type of alert to be rendered. One of Alert.Opts.Types.ALERT, Alert.Opts.Types.ERROR, Alert.Opts.Types.WARNING, Alert.Opts.Types.ADVISORY, Alert.Opts.Types.INFO, Alert.Opts.Types.SUCCESS, Alert.Opts.Types.CUSTOM. Use the Alert.Opts.Types attribute of the Alert component for access to these type strings.|
+
+## Examples
+
+### Imports
+When importing the Alert component, it includes the Types object which contains a series of string constants that can be used for setting the 'type' prop for the component as shown in the examples below.
+```jsx
+import React from 'react';
+import Alert from 'terra-alert';
+```
+
+### Examples
+__________
+Basic Alert component of type 'alert' using the default title and plain text for the message content.
+```jsx
+<Alert type={Alert.Opts.Types.ALERT} >This is an alert</Alert>
+```
+or
+```jsx
+<Alert>This is an alert</Alert>
+```
+![Image](images/basicAlert.png?raw=true)
+__________
+Basic Alert component of type 'error' using the default title and plain text for the message content.
+```jsx
+<Alert type={Alert.Opts.Types.ERROR} >This is an error</Alert>
+```
+![Image](images/basicError.png?raw=true)
+__________
+Basic Alert component of type 'warning' using the default title and plain text for the message content.
+```jsx
+<Alert type={Alert.Opts.Types.WARNING} >This is a warning</Alert>
+```
+![Image](images/basicWarning.png?raw=true)
+__________
+Basic Alert component of type 'advisory' using the default title and plain text for the message content.
+```jsx
+<Alert type={Alert.Opts.Types.ADVISORY} >This is an advisory alert</Alert>
+```
+![Image](images/basicAdvisory.png?raw=true)
+__________
+Basic Alert component of type 'info' using the default title and plain text for the message content.
+```jsx
+<Alert type={Alert.Opts.Types.INFO} >This is an information alert</Alert>
+```
+![Image](images/basicInformation.png?raw=true)
+__________
+Basic Alert component of type 'success' using the default title and plain text for the message content.
+```jsx
+<Alert type={Alert.Opts.Types.SUCCESS} >This is a success alert</Alert>
+```
+![Image](images/basicSuccess.png?raw=true)
+__________
+Alert of type 'custom' with custom title, status color, and icon, and includes HTML in message content.
+```jsx
+<Alert type={Alert.Opts.Types.CUSTOM} 
+       title="Help!" 
+       customStatusColor="orange" 
+       customIcon={<IconHelp height="1.3333rem" width="1.3333rem" />} >
+  <span>This is a <u>custom</u> alert</span>
```

Comment:
Nit - underline word makes example look like a link, maybe change to italics:
`<span>This is a <i>custom</i> alert</span>`